cli/common.h

```diff
@@ -62,10 +62,22 @@ enum switchtec_fw_type check_and_print_fw_image(int img_fd,
 			" * an index (0, 1, 2)\n" \
 			" * a PCI address (3:00.1)\n" \
 			" * an I2C path with slave address (/dev/i2c-1@0x20)\n" \
-			" * a UART path (/dev/ttyUSB0)\n" \
 	}
 
-#define DEVICE_OPTION DEVICE_OPTION_NO_PAX, \
+#define DEVICE_OPTION \
+	{ \
+			"device", .cfg_type=CFG_CUSTOM, .value_addr=&cfg.dev, \
+			.argument_type=required_positional, \
+			.custom_handler=switchtec_handler, \
+			.complete="/dev/switchtec*", \
+			.env="SWITCHTEC_DEV", \
+			.help="Switchtec device to operate on. Can be any of:\n" \
+			" * a device path (/dev/switchtec0)\n" \
+			" * an index (0, 1, 2)\n" \
+			" * a PCI address (3:00.1)\n" \
+			" * an I2C path with slave address (/dev/i2c-1@0x20)\n" \
+			" * a UART path (/dev/ttyUSB0)\n" \
+	}, \
```

Comment:
Seems to me like we just cleaned this up so it's not duplicating some much junk...

If the only thing that's different is the help message, you should still be able to use some macros to not replicate it.

Reply:
Updated to clear up the duplicates.